### ISSUE TITLE
Broken in Pharo 9 - Calypso dependencies

### ISSUE THREAD

**Issue by tblanchard** [dionisiydk/TelePharo#44]
Seems like it would be a great tool to debug a Seaside app running in a docker container in production...but it has a dependency on Calypso.

**Comment by MarcusDenker** [comment]
Yes, and it needs some more work in Pharo9 in general (and even more in Pharo10)
